tools/io_tracer_parser_tool.cc

```diff
@@ -42,28 +42,29 @@ void IOTraceRecordParser::PrintHumanReadableIOTraceRecord(
      << record.access_timestamp << ", File Operation: " << std::setw(18)
      << std::left << record.file_operation.c_str()
      << ", Latency: " << std::setw(9) << std::left << record.latency
-     << ", IO Status: " << record.io_status.c_str();
-
-  switch (record.trace_type) {
-    case TraceType::kIOGeneral:
-      break;
-    case TraceType::kIOFileNameAndFileSize:
-      ss << ", File Size: " << record.file_size;
-      FALLTHROUGH_INTENDED;
-    case TraceType::kIOFileName: {
-      if (!record.file_name.empty()) {
-        ss << ", File Name: " << record.file_name.c_str();
-      }
-      break;
+     << ", IO Status: " << record.io_status.c_str()
+     << ", File Name: " << record.file_name.c_str();
+
+  /* Read remaining options based on io_op_data set by file operation */
```

Comment:
Better to add a little bit more comments to explain the logic here, and also how to add other operations in the future.